extensions/m_webirc.c

```diff
@@ -121,6 +123,26 @@ mr_webirc(struct Client *client_p, struct Client *source_p, int parc, const char
 		return 0;
 	}
 
+	if (parc >= 6)
+	{
+		char *s;
+		for (s = strtok(parv[5], " "); s != NULL; s = strtok(NULL, " "))
```

Comment:
`parv[5]` is a `const char *` that's being passed to `strtok()`, which modifies its first argument. We probably shouldn't do that.